airflow/timetables/base.py

```diff
@@ -94,6 +95,15 @@ def logical_date(self) -> DateTime:
 class Timetable(Protocol):
     """Protocol that all Timetable classes are expected to implement."""
 
+    @cached_property
+    def interval_description(self) -> Optional[str]:
+        """Override to describe the interval.
+
+        For cron ``'30 21 * * 5'``, description could be like ``'At 09:30 PM, only on Friday'``.
+        This is used in the web UI.
+        """
+        return None
```

Comment:
```suggestion
    interval_description: Optional[str] =  None
    """Override to describe the interval.

    For cron ``'30 21 * * 5'``, description could be like ``'At 09:30 PM, only on Friday'``.
    This is used in the web UI.
    """
```
This is a protocol, so I don't reason see why we should use cached_propertty.

Reply:
And there's really no point to cache this even in subclasses either. The property is easy to calculate, and generally only accessed once in web UI to render the value.

Reply:
Also I feel this can simply be called `description`.

Reply:
implemented